google/resource_source_repos_repository.go

```diff
@@ -0,0 +1,108 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/sourcerepo/v1"
+	"log"
+)
+
+func resourceSourceReposRepository() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSourceReposRepositoryCreate,
+		Read:   resourceSourceReposRepositoryRead,
+		Delete: resourceSourceReposRepositoryDelete,
+		//Update: not supported,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"project": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceSourceReposRepositoryCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	if _, ok := d.GetOk("project"); ok {
+		log.Printf(d.Get("project").(string))
+	}
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	repoName := d.Get("name").(string)
+	name := buildRepositoryName(project, repoName)
+
+	repo := &sourcerepo.Repo{
+		Name: name,
+	}
+
+	project = "projects/" + project
```

Comment:
Change for parent := "projects/" + project to use the same nomenclature than the official docs.